### PR TITLE
Document the repo/workspace

### PR DESCRIPTION
**Pull request by Swatinem**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,93 @@ An attempt at creating some generic Rust bindings to use within Sentry.
 
 See also [RFC119](https://github.com/getsentry/rfcs/pull/119) for more details.
 
-## How To use?
+## How to write Bindings
 
-For now, do a `maturin build`, and then in your `sentry` install:
+- If you want to expose existing functionality within a `crates.io` crate:
```
Comment (loewenheim) [inline README.md:9]: This sentence initially confused me, I think it's clearer this way:
```suggestion
- If you want to expose functionality from an existing crate:
```

**Review by loewenheim — COMMENTED** [review]
Very thorough!

File: README.md [hunk 2]
```diff
@@ -4,13 +4,93 @@ An attempt at creating some generic Rust bindings to use within Sentry.
...
+
+We do not maintain any kind of `semver` guarantees whatsoever.
+For this reason, one can just increment the minor version for every new release / publish.
+
+- Manually trigger the [`Release workflow`](actions/workflows/release.yml)
+- This should ideally auto-approve the publish, publish to `pypi` and then uplift that to our
+  internal `pypi` mirror as well.
+- If the publish fails, which it unfortunately does way too often, search for a soundproof booth
+  and scream from the top of your lungs, then go to `#z-vent` and complain loudly about
+  the broken publishes and that we can’t have nice things.
```
Comment (Swatinem) [inline README.md:33]: image

Comment (loewenheim) [reply README.md:33]: 77099e99060209eda33a58858fd7b962

**Pull request merged**